pages/index.js

```diff
@@ -1,77 +1,296 @@
+    const isFirefox = ua.browser.name === 'Firefox';
+    const isLinux = ua.os.name !== 'Mac OS' && ua.os.name !== 'Windows';
+
+    this.setState({
+      desktopDownloads: [
+        {
+          icon: 'apple',
+          title: 'macOS',
+          primary: ua.os.name === 'Mac OS',
+          fileName: 'Buttercup-{version}.dmg'
+        },
+        {
+          icon: 'windows',
+          title: 'Windows',
+          primary: ua.os.name === 'Windows',
+          fileName: 'buttercup-setup-{version}.exe'
+        },
+        {
+          icon: 'linux',
+          title: 'Linux .deb',
+          primary: isLinux,
+          fileName: 'buttercup_{version}_amd64.deb'
+        },
+        {
+          icon: 'linux',
+          title: 'Linux .rpm',
+          primary: isLinux,
+          fileName: 'buttercup-{version}.x86_64.rpm'
+        }
+      ],
+      browserDownloads: [
+        {
+          icon: 'chrome',
+          title: 'Google Chrome',
+          primary: isChrome,
+          onClick: e => {
+            e.preventDefault();
+            if (isChrome) {
+              return installChromeExtension();
+            }
+            window.open(chromeExtensionUrl);
+          }
+        },
+        {
+          icon: 'firefox',
+          title: 'Mozilla Firefox',
+          primary: isFirefox,
+          onClick: e => {
+            e.preventDefault();
+            if (isFirefox) {
+              return installFirefoxExtension();
+            }
+            window.open(firefoxExtensionUrl);
+          }
+        }
+      ]
+    });
+
+    fetch('https://api.github.com/repos/buttercup/buttercup-desktop/tags')
+      .then(res => res.json())
+      .then(res => {
+        this.setState({
+          version: res[0].name
+        });
+      });
+  }
+
+  render() {
+    return (
+      <Page>
+        <section className="hero is-primary is-medium is-bold">
+          <div className="container">
+            <div className="hero-body has-text-centered">
+              <h1 className="title is-1">Buttercup</h1>
+              <h2 className="subtitle is-3">The Password Manager You Deserve.</h2>
+            </div>
+          </div>
+        </section>
+        <section className="hero is-light">
+          <div className="container">
             <div className="hero-body">
-                <div className="container has-text-centered">
-                    <h1 className="title is-1">Buttercup</h1>
-                    <h2 className="subtitle is-3">The Password Manager You Deserve.</h2>
+              <section className="columns has-text-centered">
+                <div className="column">
+                  <span className="icon is-large">
+                    <span className="fa fa-lock fa-3x" />
+                  </span>
+                  <h4 className="title is-4">Secure</h4>
+                  <p className="subtitle">
+                    Strong 256bit AES encrypted archives that meet today's security standards. Rest assured that your
```

Comment:
F*#@ing github, really.. 😂

![image](https://user-images.githubusercontent.com/3869469/31579221-5c2edc5a-b13a-11e7-9ebf-f3ec4da46d11.png)

Followed this rabbit hole for like 10 minutes..

Reply:
No fix in sight, though..

Reply:
hahaha